conans/model/conf.py

```diff
@@ -67,6 +67,10 @@ def __repr__(self):
 
     @property
     def value(self):
+        if self._value_type is list and _ConfVarPlaceHolder in self._value:
+            v = self._value[:]
+            v.remove(_ConfVarPlaceHolder)
```

Comment:
This was an uncaught bug with lists, because lists were not enough tested, right? 🍔 😄

Reply:
Yep 😢